content/en/tracing/profiler/profiler_troubleshooting.md

```diff
@@ -121,5 +128,77 @@ Override templates let you specify profiling properties to override. However, th
 
 {{< partial name="whats-next/whats-next.html" >}}
 
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Python 3.9.5)
+
+[1]: /tracing/troubleshooting/#tracer-debug-logs
+[2]: /help/
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="go" >}}
+
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Go 1.16.5)
+
+[1]: /tracing/troubleshooting/#tracer-debug-logs
+[2]: /help/
+
+## Further Reading
```

Comment:
Remove this section, duplicated with below Further Reading section

Reply:
Indeed. I copy pasted this as I was struggling to get it working. If I move this to the bottom of the file and remove the copy-pasted copies, the header rendering breaks:

![Screen Shot 2021-06-18 at 16 09 14](https://user-images.githubusercontent.com/2785847/122582055-9c683180-d04f-11eb-9e50-384c91e511f5.png)

Can you help me fix this?

Reply:
Just pushed a fix! Further reading goes at the bottom of the page after the language shortcodes.

If there is any information that is to be shared between languages (e.g. and overview or intro), you can add it to the beginning of the file and then organize each language's specific content within its appropriate shortcode. Let me know if you have any other questions.

Reply:
Thanks... But I'm a bit confused, the preview still now has the broken header rendering issue I pointed out, is that expected?